### DISCUSSION TITLE
Domains Issue

### DISCUSSION THREAD

**Discussion by sudhamrit** [vercel/community#2446] [Help] [answered]
### Summary

Another Vercel account is using this domain. Please set the following TXT record on _vercel.sunderdairy.in to use sunderdairy.in in this project. If you wish to [transfer](https://vercel.com/guides/how-can-i-move-a-domain-to-a-team#move-button) the entire domain, this will need to be done by the current domain owner. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.

![image](https://github.com/vercel/community/assets/131451608/870e7d14-85b7-483a-814f-4a9e0a61f25a)

**Comment by sudhamrit** [comment]
How can resolve this error?

**Reply by smaeda-ks** [reply]
@sudhamrit So do you actually own the domain `sunderdairy.in`? Then you need to follow that instruction to get it back to your account

**Reply by sudhamrit** [reply]
Yes, i am owner of this domain.

**Reply by sudhamrit** [reply]
Can you please share the instructions so that I can follow the steps to fix the problem.

**Reply by smaeda-ks** [reply]
@sudhamrit It looks like this domain was previously held by another Vercel account (perhaps, already deleted)? Do you recall the account name?

**Comment by sudhamrit** [comment]
@smaeda-ks the domain was previously held by another Vercel account and Domain has been already deleted.

**Reply by smaeda-ks** [reply, marked as answer]
@sudhamrit ok, that's the problem then. Please make sure to "Move" any domains to another Vercel account before you delete account/team next time. I've moved the `sunderdairy.in` to your account, please confirm (via https://vercel.com/dashboard/domains).

**Comment by Amitsananes** [comment]
have the same issue.
The domain was moved from one vercel account to my account.
i've set a dns record type TXT as instructed but i still get the same error message and waiting for verification.
even though Vercel has a disclaimer of 24-48 hours dns propagation time period, in my experience, it should take a few minutes...

**Reply by smaeda-ks** [reply]
@Amitsananes what's your domain?

**Reply by Amitsananes** [reply]
@smaeda-ks  i'm trying to verify content.rego.co.il
for now i'm using https://rego-panel-zeta.vercel.app/ as a default domain but i want to use the first one.

**Reply by smaeda-ks** [reply]
@Amitsananes the previous owner (Vercel account) still has a project there that is holding the exact same domain `content.rego.co.il`, so that's why it needs verification. Please reach out to the previous owner and ask them to release the domain from the project. Then you can delete/re-add it to your project to get it around

**Reply by Amitsananes** [reply]
i'll try talking to the other account owner and updated if the matter is resolved.
Thank you!
